extensions/wrapper/wrapper-api/src/main/java/de/sovity/edc/ext/wrapper/api/ui/model/ContractAgreementCard.java

```diff
@@ -59,4 +59,10 @@ public class ContractAgreementCard {
 
     @Schema(description = "Contract Agreement's Transfer Processes", requiredMode = Schema.RequiredMode.REQUIRED)
     private List<ContractAgreementTransferProcess> transferProcesses;
+
+    @Schema(description = "Contract Agreement's Status", requiredMode = Schema.RequiredMode.REQUIRED)
+    private String status;
+
+    @Schema(description = "Contract Agreement's Metadata", requiredMode = Schema.RequiredMode.NOT_REQUIRED)
+    private ContractAgreementMetadata contractAgreementMetadata;
```

Comment:
contractTerminationInfo

Reply:
`terminationInformation` we're already on a contract agreement object.